pinot-spi/src/main/java/org/apache/pinot/spi/data/Schema.java

```diff
@@ -185,9 +185,13 @@ public void setTimeFieldSpec(TimeFieldSpec timeFieldSpec) {
   public void addField(FieldSpec fieldSpec) {
     Preconditions.checkNotNull(fieldSpec);
     String columnName = fieldSpec.getName();
-    Preconditions.checkNotNull(columnName);
+    Preconditions.checkNotNull(columnName,
```

Comment:
(code format) If you apply the [Pinot style](https://docs.pinot.apache.org/developers/developers-and-contributors/code-setup#setup-ide), it should be formatted as following:
```
    Preconditions.checkNotNull(columnName, "Schema spec is missing `columnName` property for one of the fields.");
    Preconditions.checkState(!_fieldSpecMap.containsKey(columnName),
        "Field spec already exists for column: " + columnName);
    Preconditions.checkArgument(fieldSpec.getDataType() != null, "'%s' field is missing 'dataType' property",
        columnName);
```